src/amo/components/SearchResult/index.js

```diff
@@ -70,12 +72,17 @@ export class SearchResultBase extends React.Component<InternalProps> {
       addonInstallSource,
       clientApp,
       i18n,
+      onImpression,
```

Comment:
Are we passing this (and onClick) somewhere already, or is this scaffolding for future work?

Reply:
Scaffolding for future work. It's not being passed by anything currently.